stdlib/public/Observation/Sources/Observation/ThreadLocal.swift

```diff
@@ -0,0 +1,28 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+@_silgen_name("_swift_observation_tls_get")
+func _tlsGet() -> UnsafeMutableRawPointer?
+
+@_silgen_name("_swift_observation_tls_set")
+func _tlsSet(_ value: UnsafeMutableRawPointer?)
+
+@available(SwiftStdlib 5.9, *)
+struct ThreadLocal {
```

Comment:
These likely should instead use the existing infra in: stdlib/public/core/ThreadLocalStorage.swift

Reply:
I need TLS keys that are specific - that set of APIs from what I can tell does not do that.

Reply:
Then add API over there rather than a new parallel world here perhaps?